src/css/base/variables/_colors.scss

```diff
@@ -1,102 +1,209 @@
+  --color-typography-panel-grid-link: var(--color-ebony-clay);
+  --color-typography-sidebar-button: var(--color-mine-shaft);
+  --color-typography-table-body: var(--color-black);
+  --color-typography-table-header: var(--color-ebony-clay);
+  --color-typography-tabs-title-active: var(--color-dodger-blue);
+  --color-typography-tabs-title: var(--color-black-soft);
+  --color-typography-toc-link-active: var(--color-ebony-clay);
+  --color-typography-toc-link: var(--color-cadet-blue);
+  --color-typography-topic-grid-text: var(--color-san-marino);
+  --color-typography-topic-grid-title: var(--color-ebony-clay);
+  --color-typography-topic-link: var(--color-piction-blue);
+  --color-typography-topic-text: var(--color-black);
+  --color-typography-topic-title: var(--color-ebony-clay);
+  --color-typography-search-box-input: rgba(80, 80, 80, 0.5);
+  --color-typography-search-box-input-placeholder: rgba(80, 80, 80, 0.5) !important;
+  --color-typography-admonition-caution: var(--color-web-orange);
+  --color-typography-admonition-note: var(--color-denim);
+  --color-typography-admonition-tip: var(--color-apple);
+  --color-typography-admonition-warning: var(--color-cinnabar);
+
+  /* Color Filter */
+  --color-filter-sidenav-toctree: invert(38%) sepia(71%) saturate(6789%) hue-rotate(231deg) brightness(90%) contrast(95%);
+
+}
+
+
+.dark {
+  --dark-navigation-base: rgba(251, 251, 255, 0.04);
+  --dark-navigation-one: rgba(22, 22, 22, 0.04);
+  --dark-navigation-base-solid: rgb(41, 45, 62);
+  --dark-navigation-solid: #16212B;
+  --dark-typography-title: #F5F5FF;
+  --dark-typography-title-link: rgba(36, 213, 242, 0.9);
+  --dark-typography-paragraph: #9C9C9C;
+  --dark-surface-base: #09090A;
+  --dark-surface-one: #161616;
+  --dark-surface-two: #1A1A1B;
+  --dark-surface-three: #1C1C1D;
+  --dark-surface-four: #1F1F20;
+  --dark-surface-five: #242425;
+
+
+  --color-black-soft: rgba(255, 255, 255, 0.56);
+  --color-cadet-blue: var(--dark-typography-title-link);
+  --color-dodger-blue: var(--dark-typography-title-link);
+  --color-ebony-clay: var(--dark-typography-title);
+  --color-martinique: var(--dark-typography-title-link);;
+  --color-royal-blue: var(--dark-typography-title-link);
+  --color-san-marino: var(--dark-typography-title-link);
+
+  /** Base */
+  --color-background-base: var(--dark-typography-title);
+  --color-typography-base-link: var(--dark-typography-title-link);
+  --color-typography-base: var(--dark-typography-title);
+  --color-typography-on-this-page: var(--dark-typography-title-link);
+
+  /* Footer */
+  --color-typography-footer-link: var(--dark-typography-title);
+  --color-typography-footer-icon: var(--dark-typography-title);
+  --color-background-footer: var(--dark-navigation-solid);
+
+  /* Hero */
+  --color-background-hero: var(--dark-navigation-base);
+  --color-typography-hero-cta: var(--dark-typography-title);
```

Comment:
Here, we are organizing colors per component, whereas in the regular mode, we are organizing them by the property we are editing (Typography, Background, Border) and by component. I don't have a strong preference between these approaches, but let's be consistent.